### ISSUE TITLE
make standard format ignore all .gitignore patterns just like standard does

### ISSUE THREAD

**Issue by yoshuawuyts** [maxogden/standard-format#73]
It appears `standard-format` doesn't ignore `node_modules` dirs other than the one in the project root. This causes issues for [`linklocal`](https://www.npmjs.com/package/linklocal) users, who use local modules to build their project. `standard` does ignore these directories. It would be neat if `standard-format` handled this case similarly. Thanks!

## dir structure
```txt
.
  - index.js
  - node_modules (ignored)
  - my-module
    - node_modules (not ignored)
```

**Comment by Flet** [comment]
Hmm, is this due to `standard` including ignore patterns in `.gitignore`?

In `standard` it only seems to explicitly ignore `node_modules/**` ([index.js:23](https://github.com/feross/standard/blob/master/index.js#L23)) which matches `standard-format`.

Is the `.gitignore` in your project influencing `standard`?

**Comment by Flet** [comment]
This issue may may instead be "make standard format ignore all .gitignore patterns just like standard does" :)

**Comment by bcomnes** [comment]
Gah, using standard-format 98% of the time through an editor plugin, I totally missed this

**Comment by yoshuawuyts** [comment]
Might have been that I had an old version of standard installed. It appears that since a few weeks `standard` started linting nested `node_modules` dirs again. - https://github.com/feross/standard/issues/61#ref-commit-9c8017a. However, if `standard-format` follows `.gitignore` rules it'll be easy to fix my issues.

@bcomnes yeah, me too. I'm integrating it into an `npm script` so it's easier for collaborators to format their changes when submitting PRs.

**Comment by feross** [comment]
This is fixed in standard-format 1.5.0.

**Comment by bcomnes** [comment]
> I'm integrating it into an npm script so it's easier for collaborators to format their changes when submitting PRs.

If it works out well, add a note about this technique to the README
